.circleci/config.yml

```diff
@@ -58,17 +50,14 @@ commands:
     description: install right versions of go, docker, kubectl, and also build
     steps:
       - run:
-          #TODO add a comment saying why is this needed.
           name: Export environment variables persistent in execution shell
           command: |
             echo 'export KUBECONFIG=/home/circleci/.kube/config' >> $BASH_ENV
             echo 'export GOROOT=/usr/local/go' >> $BASH_ENV
             echo 'export GOPATH=$HOME/go' >> $BASH_ENV
             echo 'export PATH=$GOPATH/bin:$GOROOT/bin:$PATH' >> $BASH_ENV
-            echo 'export SKUPPER_SERVICE_CONTROLLER_IMAGE=${SERVICE_CONTROLLER_IMAGE_FQDN}:${CIRCLE_SHA1}' >> $BASH_ENV
-            echo 'export SKUPPER_SITE_CONTROLLER_IMAGE=${SITE_CONTROLLER_IMAGE_FQDN}:${CIRCLE_SHA1}' >> $BASH_ENV
-            echo 'export SERVICE_CONTROLLER_IMAGE=${SERVICE_CONTROLLER_IMAGE_FQDN}:${CIRCLE_SHA1}' >> $BASH_ENV
-            echo 'export SITE_CONTROLLER_IMAGE=${SITE_CONTROLLER_IMAGE_FQDN}:${CIRCLE_SHA1}' >> $BASH_ENV
+            echo 'export SKUPPER_SERVICE_CONTROLLER_IMAGE=<< pipeline.parameters.ci_service_controller_image >>' >> $BASH_ENV
+            echo 'export SKUPPER_SITE_CONTROLLER_IMAGE=<< pipeline.parameters.ci_site_controller_image >>' >> $BASH_ENV
```

Comment:
There is a trick with the environment here, the "skupper" binary will deploy the  ${SKUPPER_SERVICE_CONTROLLER_IMAGE}, but the "makefile" will build and push "${SERVICE_CONTROLLER_IMAGE}